apps/fluent-tester/src/TestComponents/CornerRadius/CornerRadiusTest.tsx

```diff
@@ -74,7 +74,7 @@ const cornerRadiusSections: TestSection[] = [
 
 export const CornerRadiusTokensTest: React.FunctionComponent = () => {
   const status: PlatformStatus = {
-    win32Status: 'Experimental',
+    win32Status: 'Beta',
     uwpStatus: 'Experimental',
     iosStatus: 'Experimental',
     macosStatus: 'Experimental',
```

Comment:
I feel like assigning the token test pages (corner radius, stroke width, spacing) a status might be confusing, since they aren't components like the other test pages. Maybe they can be N/A for all the platforms, or we can remove the status altogether? As a stretch goal was also thinking maybe there should be a separate section of the tester for the token pages

Reply:
Tokens can be used by partners though, and this page could serve as a way to tell people the status of various tokens across the FURN platforms.

Reply:
Also leaving it beta under "Production" for a V1 component which we've finished and has been out for at least a few months. Since tokens were just changed and added, I would like some bake time (2 mo ish) before calling it prod.